etc/profile-a-l/device-flasher.linux.profile

```diff
@@ -0,0 +1,82 @@
+# Firejail profile for device-flasher.linux
+# Description: CalyxOS' device flasher
+# This file is overwritten after every install/update
+
+# Usage: run firejail ./device-flasher.linux in the folder with it and your factory image.
+
+# Warning!
+# ADB does not work under this sandbox, this is not a problem however, it just means you need to
+# reboot the device into bootloader mode first. To do this, hold Volume Down + Power until you get
+# into a screen with an Android robot being repaired. You'll need OEM Unlocking beforehand.
+
+# Fedora: you may need to manually install a third-party resource for udev rules, as Fedora ships broken ones on android-tools.
+
+quiet
+include device-flasher.linux.local
+# Persistent local customizations
+include globals.local
+# Persistent global definitions
+
+ignore noexec ${HOME}
+
+include allow-bin-sh.inc
+
+blacklist /opt
+blacklist /srv
+blacklist ${RUNUSER}
+blacklist /usr/libexec
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-proc.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-x11.inc
+include disable-xdg.inc
+
+whitelist ${DOWNLOADS}
+# The Downloads folder is the only folder available to the flasher, use it or manually whitelist another.
+# Preferably use a sub-folder such as Downloads/CalyxOS/ to prevent issues.
+
+include whitelist-run-common.inc
```

Comment:
```suggestion
whitelist ${DOWNLOADS}

include whitelist-common.inc
include whitelist-run-common.inc
```

The comment is kind of redundant, as it generally applies to any profile with
`whitelist ${DOWNLOADS}` (or another XDG user directory).

Usually whitelist-common.inc is included when `whitelist ${HOME}/...` is used.

Reply:
I don't see much reason to use whitelist-common.inc, it isn't needed at all, especially being a CLI tool.

(I'll go ahead soon enough and apply the changes requested, I'd like to confirm flashing works, this is the only one I don't see why use whitelist-common in specific, removing the comments is fine)